openedx/core/djangoapps/commerce/utils.py

```diff
@@ -38,6 +39,27 @@ def get_ecommerce_api_base_url():
     return configuration_helpers.get_value('ECOMMERCE_API_URL', settings.ECOMMERCE_API_URL)
 
 
+def ecommerce_api_client(user, session=None):
+    """
+    Returns an E-Commerce API client setup with authentication for the specified user.
+
+    DEPRECATED: To be replaced with get_ecommerce_api_client.
+    """
+    claims = {'tracking_context': create_tracking_context(user)}
+    scopes = [
+        'user_id',
+        'email',
+        'profile'
+    ]
+    jwt = create_jwt_for_user(user, additional_claims=claims, scopes=scopes)
+
+    return EdxRestApiClient(
```

Comment:
`EdxRestApiClient` has been deprecated and we are removing it from our code base. Can not we use `get_ecommerce_api_client` or fix the issue in it if there is any?

Reply:
@muneebGH @bilaltahir21 ^^